api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3009,24 +3010,27 @@ public void removeObject(String bucketName, String objectName)
           private Result<DeleteError> error;
           private Iterator<DeleteError> errorIterator;
           private boolean completed = false;
-          private Iterator<String> objectNameIter = objectNames.iterator();
+          private final int batchSize = 1000;
```

Comment:
you could hard-code `1000` i.e. no need of `batchSize` variable.

Reply:
ok. done.